### PR TITLE
Style guide updates for change to SKOS annotations. Fixes #377

### LINKED ISSUE #377: Update gist style guide to reflect new annotation usage
See #351. This should be done in concert with #351.

### PR DESCRIPTION
**Pull request by rjyounes**

### CODE REVIEW

File: docs/gistStyleGuide.md [hunk 1]
```diff
@@ -63,6 +63,48 @@ Naming
...
 
+### Annotations
+
+gist uses SKOS annotations rather than `rdfs:label` and `rdfs:comment`. The accepted annotations, intended use, and previous usage are shown in the following tables. Refer to the [SKOS ontology](http://www.w3.org/2004/02/skos/core) for formal definitions.
+
+*Preferred SKOS annotations*
+
+| Annotation | Use | Replaces |
+| ---------: | --- |:---------|
+| `skos:prefLabel` | Primary label | `rdfs:label` |
```
Comment (uscholdm) [inline docs/gistStyleGuide.md:74]: Shoulds be Preferred label, not Primary label.

File: docs/gistStyleGuide.md [hunk 2]
```diff
@@ -63,6 +63,48 @@ Naming
 
 We adopt sentence over title case because the latter, while technically well-defined, has more complex rules and can introduce inconsistencies when implemented by different users.
 
+### Annotations
+
+gist uses SKOS annotations rather than `rdfs:label` and `rdfs:comment`. The accepted annotations, intended use, and previous usage are shown in the following tables. Refer to the [SKOS ontology](http://www.w3.org/2004/02/skos/core) for formal definitions.
+
+*Preferred SKOS annotations*
+
+| Annotation | Use | Replaces |
```
Comment (uscholdm) [inline docs/gistStyleGuide.md:72]: 'Replaces' is not quite right. What we mean is 'use instead of'

File: docs/gistStyleGuide.md [hunk 3]
```diff
@@ -63,6 +63,48 @@ Naming
...
+| ---------: | --- |:---------|
+| `skos:prefLabel` | Primary label | `rdfs:label` |
+| `skos:altLabel`  | Alternative label, where relevant | n/a |
+| `skos:definition` | Definition | `rdfs:comment` |
+| `skos:scopeNote` | Additional clarifying comments about the meaning or usage of a term | `rdfs:comment` |
+| `skos:example`   | One or more examples  | `rdfs:comment` |
+| `skos:editorialNote` | Notes for future editors | `rdfs:comment` |
```
Comment (uscholdm) [inline docs/gistStyleGuide.md:79]: Notes for current editors too. Maybe just editors?  Pico-point.

**Review by uscholdm — APPROVED** [review]
This is good. I left a couple of suggestions, heed or ignore as you see fit.

File: docs/gistStyleGuide.md (hunk 3 above)
Comment (rjyounes) [reply docs/gistStyleGuide.md:79]: I agree. Fixed.

**Comment by rjyounes** [comment]
Good catches, I've made the updates.

**Pull request merged**